Program.cs

```diff
@@ -3,17 +3,48 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to sort and swap special characters.
+// Given a char array, a special character S and sort order (default order Ascending), print sorted array keeping elements matching S at the last.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Sort and Swap Array</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method displays the sorted and swapped array</summary>
+      static void Main () {
+         string letters = GetValidInput ("Enter the characters (only alphabets) to be sorted (as string): ");
+         string splCharInput = GetValidInput ("Enter the special character to be swapped to the end: ", false);
+         char splChar = splCharInput[0];

```

Comment:
You can directly assign the char in the first line itself:
`char splChar = GetValidInput (......)[0];`